plugins/polio/js/src/forms/FormAForm.tsx

```diff
@@ -0,0 +1,159 @@
+            });
+        }
+    }, [accessor, fieldValues, setFieldTouched]);
+
+    // // Set TextFields values to null if empty to avoid 400.
+    useEffect(() => {
+        if (fieldValues?.forma_usable_vials === '') {
+            setFieldValue(`${accessor}.forma_usable_vials`, null);
+        }
+        if (fieldValues?.forma_unusable_vials === '') {
+            setFieldValue(`${accessor}.forma_unusable_vials`, null);
+        }
+        if (fieldValues?.forma_missing_vials === '') {
+            setFieldValue(`${accessor}.forma_missing_vials`, null);
+        }
+    }, [
+        accessor,
+        fieldValues?.forma_missing_vials,
+        fieldValues?.forma_unusable_vials,
+        fieldValues?.forma_usable_vials,
+        setFieldValue,
+    ]);
+
+    const disableComment = !(
+        fieldValues?.forma_missing_vials &&
+        fieldValues?.forma_unusable_vials &&
+        fieldValues?.forma_usable_vials &&
+        fieldValues?.forma_date &&
+        fieldValues?.forma_reception
+    );
+
+    return (
+        <>
+            <Grid container direction="row" item xs={12} spacing={2}>
+                <Grid item lg={3} md={6}>
+                    <Field
+                        label={formatMessage(MESSAGES.formAReception)}
+                        name={`${accessor}.forma_reception`}
+                        component={DateInput}
+                        className={classes.input}
+                    />
+                </Grid>
+                <Grid item lg={3} md={6}>
+                    <Field
+                        label={formatMessage(MESSAGES.formAUnusableVials)}
+                        name={`${accessor}.forma_unusable_vials`}
+                        component={DebouncedTextInput}
+                        debounceTime={300}
+                        className={classes.input}
+                    />
+                </Grid>
+                <Grid item lg={3} md={6}>
+                    <Field
+                        label={formatMessage(MESSAGES.formAMissingVials)}
+                        name={`${accessor}.forma_missing_vials`}
+                        component={DebouncedTextInput}
+                        debounceTime={300}
+                        className={classes.input}
+                    />
+                </Grid>
+                <Grid item lg={3} md={6}>
+                    <Field
+                        label={formatMessage(MESSAGES.formAUsableVials)}
+                        name={`${accessor}.forma_usable_vials`}
+                        component={DebouncedTextInput}
+                        debounceTime={300}
+                        className={classes.input}
+                    />
+                </Grid>
+            </Grid>
+            <Grid
+                container
+                direction="row"
+                item
+                xs={12}
+                spacing={2}
+                justifyContent="flex-start"
+            >
+                <Grid item lg={3} md={6}>
```

Comment:
this is why you have too much spacing, try to put `forma_date` under `forma_reception` , `forma_comment ` under `formAUnusableVials `